components/suggest/src/yelp.rs

```diff
@@ -360,7 +360,11 @@ impl<'a> SuggestDao<'a> {
         // Otherwise, apply prefix-match.
         Ok(
             match self.conn.query_row_and_then_cachable(
-                "SELECT keyword FROM yelp_subjects WHERE keyword BETWEEN :candidate AND :candidate || x'FFFF' ORDER BY LENGTH(keyword) ASC LIMIT 1",
+                "SELECT keyword
+                 FROM yelp_subjects
+                 WHERE keyword BETWEEN :candidate AND :candidate || x'FFFF'
+                 ORDER BY LENGTH(keyword) ASC, keyword ASC
```

Comment:
The only substantive change here is to add a backstop `keyword ASC` to the `ORDER BY` to ensure a consistent ordering for keywords that are the same length.